### PR TITLE
Fix quiz templates

### PR DESCRIPTION
**Pull request by adityacp**
Changed Quiz  interface View for students

Made tracebacks proper with line number and output text area is added below program text area.
quiz_output_traceback

Changed  `Language: Python` to `Python Code Question`
quiz_view

### CODE REVIEW

File: yaksh/evaluator_tests/test_python_evaluation.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def add(a, b):
         result = get_class.evaluate(**kwargs)
         err = result.get("error").splitlines()
         self.assertFalse(result.get("success"))
-        self.assertEqual(2, len(err))
+        self.assertEqual(969, len(err))
```
Comment (prathamesh920) [inline yaksh/evaluator_tests/test_python_evaluation.py]: Will always be 969?

Comment (adityacp) [inline yaksh/evaluator_tests/test_python_evaluation.py]: for that particular test case example it is fixed

**Comment by ankitjavalkar** [comment]
Code is mergeable

File: yaksh/code_evaluator.py [hunk 2]
```diff
@@ -108,9 +108,14 @@ def safe_evaluate(self, user_answer, test_case_data, file_paths=None):
...
             err = self.timeout_msg
+        except OSError:
+            msg = traceback.format_exc(limit=0)
+            err = "Error: {0}".format(msg)
         except Exception:
-            err = "Error: {0}".format(traceback.format_exc(limit=0))
-
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            tb_list = traceback.format_exception(exc_type, exc_value, exc_tb)
+            del tb_list[1:3]
```
Comment (prabhuramachandran) [inline yaksh/code_evaluator.py]: I am not sure about this.  I don't think it is a good idea to do this and indiscriminately remove the two lines from `tb_list`.  What if tb_list has only two lines in it?  Can you pease ensure that this is also fully tested for different cases or write it a bit more safely.

File: yaksh/templates/yaksh/question.html [hunk 3]
```diff
@@ -176,10 +171,20 @@ <h3 class="brand"><strong>Online Test</h3></strong>
...
 
          <h4>Program:</h4>
-        <textarea name="answer" id="answer"></textarea> 
-<br>
+         <textarea name="answer" id="answer"></textarea>
+         <br>
 
          {% endif %}
+         {% if question.type == "code" %}
+         <br><h4>Output:</h4></br>
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/question.html]: <br/> is all that is needed, br does not need a /br like here.

File: yaksh/code_evaluator.py (hunk 2 above)
Comment (adityacp) [inline yaksh/code_evaluator.py]: I have added test cases for some exceptions and I have manually tested almost all built-in exceptions from interface also. It worked fine. 
lines which are removed from `tb_list` contains file path from which error occurred which can be removed.

Comment (prabhuramachandran) [inline yaksh/code_evaluator.py]: At least check if len(tb_list) >2 before deleting the lines?

**Pull request merged**